tools/travis/travis.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/sh
```

Comment:
PLEASE FIX:  All .sh files reequire (currently) a BSD 3-clause license.  No unlicensed shell scripts may come in.

(License will, eventually, be changed to Apache).

Reply:
i added apache license